problem_builder/mentoring.py

```diff
@@ -214,6 +225,10 @@ def author_preview_view(self, context):
         self.include_theme_files(fragment)
         return fragment
 
+    def max_score(self):
+        """ Maximum score. We scale all scores to a maximum of 1.0 so this is always 1.0 """
+        return 1.0
```

Comment:
Score vs grade: They are used interchangeably in the edX platform, but they are [trying to move to a more consistent approach](https://openedx.atlassian.net/wiki/display/TNL/edX+Grading+Guide+for+Developers) where "score" means a numeric result returned by an individual component such as an XBlock, and "grade" refers to the user's overall standing in a course (as a percentage and a letter grade or pass/fail). So the "grade" is a function of the user's scores plus the rules and weightings configured for the course.

If the user has not yet generated a score for this block:
* The `max_score` method, if defined, will result in '0/x' being displayed on the progress page for this block. As far as I know, the return value is not used directly, as the total possible score is scaled by the platform to the value of `weight`. So if `weight` is `1`, then the progress page will always display `0/1`.

If the user has generated a score for this block (which is done, confusingly, by publishing a `grade` XBlock event):
* Then the platform displays the user's score for this block as (`value / max_value * weight`), where `weight` is the field of the XBlock and `value`/`max_value` come from the event itself.